### PR TITLE
ddl: handle parallel rename table

### PR DESCRIPTION
**Pull request by zimulala**
```create table test.exists (a int);```
If we have the two SQLs as follows:
```
create table test.t (c int);
alter table test.exists rename test.t;
```
There are running concurrently. If RenameTable runs after CreateTable, it will call meta.DropTable and meta.CreateTable. Then `t(a int)` will replace `t(c int)`.
This PR handle it and unify test code

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @shenli @coocood @ciscoxll

File: ddl/ddl_db_change_test.go [hunk 1]
```diff
@@ -544,82 +582,11 @@ func (s *testStateChangeSuite) testParrallelExecSQL(c *C, sql string) {
...
+	s.testParallelExecSQL(c, "create table if not exists test_not_exists(a int);")
 }
 
 // TestCreateDBIfNotExists parallel exec create database if not exists xxx. No error returns is expected.
 func (s *testStateChangeSuite) TestCreateDBIfNotExists(c *C) {
 	defer s.se.Execute(context.Background(), "drop database test_not_exists")
-	s.testParrallelExecSQL(c, "create database if not exists test_not_exists;")
-}
-
-func (s *testStateChangeSuite) TestParallelChangeColumnName(c *C) {
```
[the remark this answers is not in the document]
Comment (zimulala) [reply ddl/ddl_db_change_test.go]: @shenli 
I don't remove this function，it is in line 446.

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@jackysp @coocood  PTAL

**Pull request merged**